TrabajoPractico1/nuevo_turno.html

```diff
@@ -31,34 +31,36 @@ <h1><img src="/imagenes/logo.jpg" alt="Enterprise Logo" width="500" height="600"
 		En este caso enlace hacia información sobre cuidados de COVID-19.
 	-->
 	<aside>
-		<a href="#"><img src="" alt="Información COVID-19"></a>	
+		<a href="#"><img src="/imagenes/covid.jpg" alt="Información COVID-19"></a>	
 	</aside>
 
 	<!--
 		Contenido principal del sitio. Contendrá el formulario para la solicitud de nuevos turnos.
 	-->
 	<main>
 			<header>
-          		<h1>Nuevo Turno </h1>
+          		<h1>Nuevo Turno</h1>
       		</header>
 		<form method="POST" name="nuevo_turno" autocomplete="on">  
 			<fieldset>
+				<legend>Información Personal</legend>
 				<label for="apenomb">Apellido y Nombre</label><br>
-				<input type="text" name="apenomb" required placeholder="Peréz Juan" autofocus tabindex="1"><br>
-				<label for="email">Email:</label><br>
-				<input type="email" name="email" placeholder="ejemplo@ejemplo.com.ar" tabindex="2">
+				<input type="text" id="apenomb" required placeholder="Peréz Juan" autofocus tabindex="1"><br>
```

Comment:
Cambio el atributo "name" por "id" porque sino el validador decía que el label tiene que tener algun input donde el "for" coincida con el "id"